### PR TITLE
Fix PageBody

### PR DESCRIPTION
**Pull request by natalynyu**
The purpose of adding a padding to `PageBody` is to solve for an issue in Course Planner in which the `Footer` covers the bottom of a table making the information in the table inaccessible to the user. By adding some padding to the bottom of `PageBody`, we are making it so that the footer does not obfuscate any information.

The purpose of adding a padding to PageBody is to solve for an issue in Course Planner in which the footer covers the bottom of a table making the information in the table inaccessible to the user. By adding some padding to the bottom of PageBody, we are making it so that the footer does not obfuscate any information.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality
      to change)

## Checklist:

- [x] I have run `eslint` on the code
- [x] I have added JSDoc for all of my code (where applicable)
- [ ] I have added tests to cover my changes.

## Priority:
- [x] Normal 
- [ ] High 

## Related Issues:

Fixes #704

### CODE REVIEW

File: src/Layout/PageBody.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ export interface PageBodyProps {
 
 const PageBody = styled.main<PageBodyProps>`
   margin: ${fromTheme('ws', 'small')};
+  padding-bottom: 40px;
```
Comment (robertmain) [inline src/Layout/PageBody.tsx:12]: I've had a look at this on info.seas.harvard.edu and it seems as though the height of the footer is obscuring the bottom of the `PageBody` component
image

Given that, setting `padding: 40px;` on the `PageBody` component as you have here does indeed fix the issue

image

However, if the footer is made taller, we're likely going to run into the same issue again.

image

Given that - I wonder if it makes sense to instead have either have optional `padding` parameter(s)? that one can pass(as well as a `height` parameter we can pass to [`Footer.tsx`](https://github.com/seas-computing/mark-one/blob/develop/src/Layout/Footer.tsx#L31) at some point in the future)

**Review by robertmain — CHANGES_REQUESTED** [review]
Thanks for doing this, Natalyn! Just a note regarding hard coding the padding value

File: src/Layout/PageBody.tsx (hunk 1 above)
Comment (natalynyu) [reply src/Layout/PageBody.tsx]: Thanks Robert, that's a really good point. I updated the `PageBody` component to have an optional parameter for `padding` to give the user more flexibility.

Comment (rmainseas) [reply src/Layout/PageBody.tsx]: Looks great. Merge at will.

**Review by rmainseas — APPROVED** [review]
Looks great! :rocket:

**Pull request merged**